### PR TITLE
Add some new data for car

### PR DESCRIPTION
**Pull request by agasigp**

### CODE REVIEW

File: src/CarData.php [hunk 1]
```diff
@@ -33,7 +33,6 @@ public static function getVehicleGearBoxType() : array
         return static::$vehicleGearBox;
     }
 
-    // phpcs:disable
```
Comment (pelmered) [inline src/CarData.php]: Why did you remove this?

File: src/CarData.php [hunk 2]
```diff
@@ -588,6 +588,4 @@ public static function getVehicleGearBoxType() : array
         'manual'    => 70,
         'automatic' => 30,
     ];
-
-    // phpcs:enable
```
Comment (pelmered) [inline src/CarData.php:845]: Why did you remove this?

File: src/CarData.php (hunk 1 above)
Comment (agasigp) [reply src/CarData.php]: It's useless. I don't see any purpose for that code, it's just commented, so I remove it. Any reason why it should not be removed?
